core/controllers/home_test.py

```diff
@@ -29,7 +29,7 @@ def test_logged_out_homepage(self):
         self.assertEqual(response.status_int, 200)
         response.mustcontain(
             'Your personal tutor',
-            'Oppia - Gallery', 'About', 'Login', no=['Logout'])
+            'Oppia - Gallery', 'About', 'Sign in', no=['Logout'])
```

Comment:
"Login" is now a substring of onLoginButtonClicked(). If you look at the page, the text has since changed to "Sign in" anyway, so this test was out of date.